### ISSUE TITLE
Fast PR Builder by running tests in parallel 

### ISSUE THREAD

**Issue by mesutcelik** [hazelcast/hazelcast#5581]
https://maven.apache.org/surefire/maven-surefire-plugin/examples/fork-options-and-parallel-execution.html

To do that, we should move MulticastTests from Quick to some other category.

**Comment by mesutcelik** [comment]
@bilalyasar ,

Can you please update this issue with your findings?

**Comment by bilalyasar** [comment]
this is already done.
